### DISCUSSION TITLE
i can't install netclient on windows

### DISCUSSION THREAD

**Discussion by alshmokh** [gravitl/netmaker#2295] [Q&A]
i can't install net client on Windows i try Powershell but its failed is there another way to install net client without complex

**Comment by dcgsteve** [comment]
You should just be able to download the latest version from [releases](https://github.com/gravitl/netclient/releases), create an enrollment key and then execute something like this in an administrator command prompt:

`netclient register -t YourEnrollemtnKeyHere`

This is the process I just did and it worked fine?
